src/controllers/cohort.js

```diff
@@ -10,3 +16,23 @@ export const create = async (req, res) => {
     return sendMessageResponse(res, 500, 'Unable to create cohort')
   }
 }
+
+/**
+ * Updates a user by ID
+ * @param {import('express').Request} req
+ * @param {import('express').Response} res
+ */
+export const getById = async (req, res) => {
+  // TODO: not User.findAll()
+  const id = parseInt(req.params.id)
```

Comment:
at the moment, there's no validation that the cohort with this id exists, not sure if you want to return 404 for non-existent cohorts